src/app/parcels/ParcelsPage.tsx

```diff
@@ -248,6 +248,7 @@ const parcelIdParam = "parcelId";
 
 const defaultNumberOfParcelsPerPage = 100;
 const numberOfParcelsPerPageOptions = [10, 25, 50, 100];
+export const nullPostcodeDisplay = "NFA";
```

Comment:
we should probably have this line in a different file as this is used in multiple places, not just the ParcelsPage. Is there anywhere sensible? We could just create a file `src/common/nullPostCodeDisplay.ts` or something

Reply:
I think putting it in common/format.ts would be sensible